src/Validator/Constraints/ItemOwnerValidator.php

```diff
@@ -41,17 +41,17 @@ public function validate($value, Constraint $constraint)
             return;
         }
 
-        if (!$constraint instanceof PostOwner) {
-            throw new UnexpectedTypeException($constraint, PostOwner::class);
+        if (!$constraint instanceof ItemOwner) {
+            throw new UnexpectedTypeException($constraint, ItemOwner::class);
         }
 
         /** @var User $user */
         $user = $this->security->getUser();
         $userRepository = $this->entityManager->getRepository(User::class);
```

Comment:
EntityManager is used only to get UserRepository. Maybe you will get UserRepository into it constructor?

Reply:
Good catch but this code does not associated with this pull request